gtfs/__main__.py

```diff
@@ -109,67 +129,137 @@ def list_feeds(
         if pretty is True:
             print("\n" + pretty_output.get_string())
 
+        if typer.confirm("Do you want to fetch feeds from these sources?"):
+            fetch_feeds(sources=filtered_srcs[:-1])
+
 
 @app.command()
-def fetch_feeds(sources=None):
-    """
+def fetch_feeds(
+    sources: Annotated[
+        Optional[str],
+        typer.Option(
+            "--sources",
+            "-src",
+            help="pass value as a string separated by commas like this: Berlin,AlbanyNy,...",
+            callback=check_sources,
+        ),
+    ] = None,
+    search: Annotated[
+        Optional[str],
+        typer.Option(
+            "--search",
+            "-s",
+            help="search for feeds based on a string",
+        ),
+    ] = None,
```

Comment:
this should be done on `list_feeds` IMO

Reply:
Yeah I had a doubt regarding that.... will do